lib/elixir/lib/io/ansi.ex

```diff
@@ -170,6 +170,29 @@ defmodule IO.ANSI do
   @doc "Sends cursor home."
   defsequence(:home, "", "H")
 
+  @doc """
+  Sends cursor to the absolute position specified by `lines`, `columns`, 
+  with 0,0 being the top left corner.
+  """
+  @spec cursor(integer, integer) :: String.t()
+  def cursor(line, column) when line >= 0 and column >= 0, do: "\e[#{line};#{column}H"
+
+  @doc "Sends cursor `lines` up. Defaults to 1"
```

Comment:
We don't need to mention the default, as `\\` is standard in Elixir. Let's just have 

```elixir
@doc "Sends cursor `lines` lines up.
```